gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -1903,7 +1904,9 @@ gst_tensor_filter_stop (GstBaseTransform * trans)
     GST_OBJECT_UNLOCK (self);
   }
 
-  gst_tensor_filter_disable_invoke_async (&priv->prop);
+  if (priv->prop.invoke_async)
```

Comment:
invoke_async can be updated at run-time by users or applications. You need to check if invoke_async thread is running.

E.g., 
```
1. configure invoke_async=true
2. start pipeline
3. configure invoke_async=false
4. stop pipeline
```